test/testinfra/MUXTestInfra/Common/InputHelper.cs

```diff
@@ -295,6 +309,74 @@ public static void TapAndHold(UIObject obj, uint durationMs)
             Wait.ForIdle();
         }
 
+        public static void DoubleTap(UIObject obj)
```

Comment:
I didn't end up using these methods, but I figured I'd keep them around since there's no harm having them available in case we need them in the future.